src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -56,7 +55,7 @@ class Connection
     private $deleteAfterReject;
     private $couldHavePendingMessages = true;
 
-    public function __construct(array $configuration, array $connectionCredentials = [], array $redisOptions = [], \Redis|\RedisCluster $redis = null)
+    public function __construct(array $configuration, array $connectionCredentials = [], array $redisOptions = [], \Redis | \RedisCluster $redis = null)
```

Comment:
should be reverted (same elsewhere)

Reply:
Okey. 
FYI: This came form fabbot

Reply:
yep, I'm looking forward to https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5734